AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -132,6 +132,7 @@ fastbox.su##.usePersonalData
 bojoko.com##.evt-notification-bottom
 onlyfans.com##.b-cookies-informer
 biblioteka.shop##.banner-cookie
+dvdvideosoft###cookie_banner
```

Comment:
We already have a generic rule, but I don't think that's issue related to the filters.